p5js/YOLO/YOLO_single_image/index.html

```diff
@@ -5,26 +5,17 @@
 	<meta http-equiv="X-UA-Compatible" content="IE=edge">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
 
-	<title>template</title>
+	<title>Yolo with image</title>
 
 	<script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.7.3/p5.min.js"></script>
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.7.3/addons/p5.dom.min.js"></script>
-  <script src="https://unpkg.com/ml5@0.2.1/dist/ml5.min.js" type="text/javascript"></script>
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.7.3/addons/p5.dom.min.js"></script>
+	<!-- <script src="https://unpkg.com/ml5@0.2.1/dist/ml5.min.js" type="text/javascript"></script> -->
+	<script src="http://localhost:8080/ml5.js"></script>
```

Comment:
Note for later: I think when we are using `localhost` it should probably be in `development` and not in `release`?

Reply:
yes! 100% agreed with this. Since this PR matches the change to the YOLO results from `className` to `label` and `probability` to `confidence` I wanted to ensure in this instance to specify localhost. 

I'll add the `development` and `release` branches to the examples repo. Thanks for the feedback!